### PR TITLE
improvement: ARSN-123 modify BucketAlreadyOwnedByYou error description

### PR DESCRIPTION
**Pull request by XinLi0207**
modify the `BucketAlreadyOwnedByYou` description to be more user-friendly.
the discussion is here: https://scality.atlassian.net/browse/ARTESCA-2237?focusedCommentId=306745

### CODE REVIEW

File: errors/arsenalErrors.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   },
   "BucketAlreadyOwnedByYou": {
     "code": 409,
-    "description": "Your previous request to create the named bucket succeeded and you already own it. You get this error in all AWS regions except US Standard, us-east-1. In us-east-1 region, you will get 200 OK, but it is no-op (if bucket exists S3 will not do anything)."
+    "description": "A bucket with this name already exists and owned by you"
```
Comment (ghost) [inline errors/arsenalErrors.json]: What do you think of:

```suggestion
    "description": "A bucket with this name exists and is already owned by you"
```

To be closer to the name of the error?

**Review by ghost — APPROVED** [review]
Agreed with William's comment, otherwise LGTM

File: errors/arsenalErrors.json (hunk 1 above)
Comment (XinLi0207) [reply errors/arsenalErrors.json]: fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**